_example/main.go

```diff
@@ -64,3 +81,86 @@ func createTestDatabase() *mem.Database {
 	table.Insert(ctx, sql.NewRow("Evil Bob", "evilbob@gmail.com", []string{"555-666-555", "666-666-666"}, time.Now()))
 	return db
 }
+
+func initMetrics() {
```

Comment:
Do we want to add all this extra code in the example? It seems like a lot of code not related to the example to have there. Put it in the README to let the user know how to customize the metrics? Yes. But here does not make much sense imho.